# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/ShadowLayers.cpp
# Commit: 063622f1a9b5
# Full Hash: 063622f1a9b5bf4f4ab425fcf2ff67140d022578
# Author: Alexandre Lissy <lissyx+mozillians@lissyx.dyndns.org>
# Date: 2021-07-15 21:52:49
# Description:
#   Bug 1719668 - Do not force XFlush() on XRender r=jrmuizel,nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119557
# ==============================================================================

diff -r f597999c46c0 -r 063622f1a9b5 gfx/layers/ipc/ShadowLayers.cpp
--- a/gfx/layers/ipc/ShadowLayers.cpp	Thu Jul 15 08:42:08 2021 +0000
+++ b/gfx/layers/ipc/ShadowLayers.cpp	Thu Jul 15 08:56:20 2021 +0000
@@ -586,13 +586,6 @@
     return true;
   }
 
-  if (!mTxn->mPaints.IsEmpty()) {
-    // With some platforms, telling the drawing backend that there will be no
-    // more drawing for this frame helps with preventing command queues from
-    // spanning across multiple frames.
-    gfxPlatform::GetPlatform()->FlushContentDrawing();
-  }
-
   MOZ_LAYERS_LOG(("[LayersForwarder] destroying buffers..."));
 
   MOZ_LAYERS_LOG(("[LayersForwarder] building transaction..."));